CRT.c

```diff
@@ -820,7 +820,121 @@ static int CRT_colorSchemes[LAST_COLORSCHEME][LAST_COLORELEMENT] = {
       [DYNAMIC_YELLOW] = ColorPair(Yellow, Black),
       [DYNAMIC_WHITE] = ColorPair(White, Black),
    },
-   [COLORSCHEME_BROKENGRAY] = { 0 } // dynamically generated.
+   [COLORSCHEME_BROKENGRAY] = { 0 }, // dynamically generated.
+   [COLORSCHEME_NORD] = {
+      [RESET_COLOR] = A_NORMAL,
+      [DEFAULT_COLOR] = A_NORMAL,
+      [FUNCTION_BAR] = ColorPair(Black, Cyan),
+      [FUNCTION_KEY] = A_NORMAL,
+      [PANEL_HEADER_FOCUS] = ColorPair(Black, Cyan),
+      [PANEL_HEADER_UNFOCUS] = A_REVERSE,
+      [PANEL_SELECTION_FOCUS] = ColorPair(Black, Cyan),
+      [PANEL_SELECTION_FOLLOW] = A_REVERSE,
+      [PANEL_SELECTION_UNFOCUS] = A_BOLD,
+      [FAILED_SEARCH] = A_REVERSE | A_BOLD | ColorPair(Yellow, Black),
+      [FAILED_READ] = A_BOLD | ColorPair(Yellow, Black),
+      [PAUSED] = A_BOLD | ColorPair(Black, Cyan),
+      [UPTIME] = A_BOLD,
+      [BATTERY] = A_BOLD,
+      [LARGE_NUMBER] = A_BOLD | ColorPair(Yellow, Black),
+      [METER_SHADOW] = A_DIM,
+      [METER_TEXT] = A_NORMAL,
+      [METER_VALUE] = A_BOLD,
+      [METER_VALUE_ERROR] = A_BOLD,
+      [METER_VALUE_IOREAD] = A_NORMAL,
+      [METER_VALUE_IOWRITE] = A_NORMAL,
+      [METER_VALUE_NOTICE] = A_BOLD | ColorPair(Cyan, Black),
+      [METER_VALUE_OK] = A_NORMAL,
+      [METER_VALUE_WARN] = A_BOLD,
+      [LED_COLOR] = A_NORMAL,
+      [TASKS_RUNNING] = A_BOLD,
+      [PROCESS] = A_NORMAL,
+      [PROCESS_SHADOW] = A_DIM,
+      [PROCESS_TAG] = A_BOLD | ColorPair(Cyan, Black),
+      [PROCESS_MEGABYTES] = A_BOLD | ColorPair(White, Black),
+      [PROCESS_GIGABYTES] = A_BOLD | ColorPair(Cyan, Black),
+      [PROCESS_BASENAME] = A_BOLD,
+      [PROCESS_TREE] = A_BOLD,
+      [PROCESS_RUN_STATE] = A_BOLD,
+      [PROCESS_D_STATE]  = A_BOLD | ColorPair(Yellow, Black),
+      [PROCESS_HIGH_PRIORITY] = A_BOLD,
+      [PROCESS_LOW_PRIORITY] = A_DIM,
+      [PROCESS_NEW] = A_BOLD,
+      [PROCESS_TOMB] = A_DIM,
+      [PROCESS_PRIV] = A_BOLD | ColorPair(Cyan, Black),
+      [BAR_BORDER] = A_BOLD,
+      [BAR_SHADOW] = A_DIM,
+      [SWAP] = A_BOLD,
+      [SWAP_CACHE] = A_NORMAL,
+      [SWAP_FRONTSWAP] = A_DIM,
+      [GRAPH_1]  = A_BOLD,
+      [GRAPH_2] = A_NORMAL,
+      [MEMORY_USED] = A_BOLD | ColorPair(Yellow, Black),
+      [MEMORY_BUFFERS] = A_NORMAL,
+      [MEMORY_BUFFERS_TEXT] = A_NORMAL,
+      [MEMORY_CACHE] = A_NORMAL,
+      [MEMORY_SHARED] = A_NORMAL,
+      [MEMORY_COMPRESSED] = A_DIM,
+      [HUGEPAGE_1] = A_BOLD,
+      [HUGEPAGE_2] = A_NORMAL,
+      [HUGEPAGE_3] = ColorPair(Black, Cyan),
+      [HUGEPAGE_4] = ColorPair(Black, Cyan),
+      [LOAD_AVERAGE_FIFTEEN] = A_DIM,
+      [LOAD_AVERAGE_FIVE] = A_NORMAL,
+      [LOAD_AVERAGE_ONE] = A_BOLD,
+      [LOAD] = A_BOLD,
+      [HELP_BOLD] = A_BOLD,
+      [HELP_SHADOW] = A_DIM,
+      [CLOCK] = A_BOLD,
+      [DATE] = A_BOLD,
+      [DATETIME] = A_BOLD,
+      [CHECK_BOX] = A_BOLD,
+      [CHECK_MARK] = A_NORMAL,
+      [CHECK_TEXT] = A_NORMAL,
+      [HOSTNAME] = ColorPair(Black, Cyan),
```

Comment:
Is this intentional? Looks kinda strange to have the hostname in the meter with background, when everything else is colored text with black background.

You can test this by adding the Hostname and System meters in the header.